go.mod

```diff
@@ -0,0 +1,43 @@
+module task
```

Comment:
Why not using Bazel?
- https://bazel.build/about/intro
- https://earthly.dev/blog/build-golang-bazel-gazelle/

Reply:
Given the relatively small scale of the project, I felt that the built-in Go build tools adequately meet my current needs. Additionally, I am already familiar with the Go build process.

In the future, if the project expands or if there is a requirement for more advanced CI/CD capabilities, I am open to exploring alternative build systems, including Bazel. Currently, if the need for a basic CI/CD pipeline arises, I find that using GitHub Actions provides sufficient flexibility.